src/bootstrap/bin/rustc.rs

```diff
@@ -185,7 +185,8 @@ fn main() {
             cmd.env("RUST_SAVE_ANALYSIS_CONFIG",
                     "{\"output_file\": null,\"full_docs\": false,\
                      \"pub_only\": true,\"reachable_only\": false,\
-                     \"distro_crate\": true,\"signatures\": false,\"borrow_data\": false}");
+                     \"distro_crate\": true,\"signatures\": false,\"borrow_data\": false,\
+                     \"reachable_only\": false}");
```

Comment:
The `reachable_only` is declared twice.

Reply:
My mistake. Started looking at an older git checkout which didn't have https://github.com/rust-lang/rust/commit/794ada06e7ad370ec48b4de69e43eb59d69543c9